packages/askar/src/wallet/AskarWallet.ts

```diff
@@ -715,12 +715,7 @@ export class AskarWallet implements Wallet {
         throw new WalletError(`No content found for record with public key: ${publicKeyBase58}`)
       }
     } catch (error) {
-      if (
-        isAskarError(error) &&
-        (error.code === AskarErrorCode.NotFound ||
-          // FIXME: this is current output from askar wrapper but does not describe specifically a not found scenario
-          error.message === 'Received null pointer. The native library could not find the value.')
-      ) {
+      if (isAskarError(error, AskarErrorCode.NotFound)) {
```

Comment:
This check can actually be removed, as we will return `null` instead of a `NotFound` error being thrown